cmd/kubeadm/app/kubeadm.go

```diff
@@ -30,6 +30,7 @@ func Run() error {
 	// We do not want these flags to show up in --help
 	pflag.CommandLine.MarkHidden("google-json-key")
 	pflag.CommandLine.MarkHidden("log-flush-frequency")
+	pflag.CommandLine.MarkHidden("fuzz-iters")
```

Comment:
Obviously, yes, was too tired yesterday when uploading this I didn't even think about it :)
Thanks for forcing me to check the import chain, now fixed